torch/onnx/_internal/diagnostics/infra/sarif/version.py

```diff
@@ -1,3 +1,5 @@
-SARIF_VERSION = "2.1.0"
-SARIF_SCHEMA_LINK = "https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/schemas/sarif-schema-2.1.0.json"
+from typing_extensions import Final
+
+SARIF_VERSION: Final = "2.1.0"
```

Comment:
Final: nice: learned something new!

Reply:
Adding type annotation to sarif is very helpful. I found myself checking the original spec much less now that I could jump around instead. And this `Final` was forced upon after the version attribute in sarif was annotated as Literal[...].